tasks/main.yml

```diff
@@ -15,12 +15,7 @@
 
 - include: dataverse-required-variables.yml
   tags:
-   - payara
-   - dataverse
-   - storage
-   - postinstall
-   - s3
-   - gui
+   - always
```

Comment:
This may have side effects, although I do not think so. On the other hand, removing rows of code is almost always a good idea. :)